packages/react/cypress/component/auto/form/AutoFormDynamicFormInputChanges.cy.tsx

```diff
@@ -12,8 +12,8 @@ describeForEachAutoAdapter("AutoForm", ({ name, adapter: { AutoForm }, wrapper }
         <AutoForm action={api.widget.update} findBy={findBy} />
 
         <div>
-          <button id="setFindById1" onClick={() => setFindBy("1")} />
-          <button id="setFindById2" onClick={() => setFindBy("2")} />
+          <button style={{ width: 1, height: 1 }} id="setFindById1" onClick={() => setFindBy("1")} />
+          <button style={{ width: 1, height: 1 }} id="setFindById2" onClick={() => setFindBy("2")} />
```

Comment:
Shouldn't buttons have a default style that makes them have dimensions and be findable? this works but i wouldn't have thought it was necessary

Reply:
Because there is no label or text inbetween the button, it won't have anything.
If we did <button>hello</button> then the button will take up the space of the `hello`